GSOC 2025 Gigaflow

AdvaySingh1 opened the pull request:
Added Gigaflow GSOC 2025 project repo.

Dscano reviewed (changes requested):
The Gigaflow Project and NextGarch Lab links are working, but all the other links listed in the report are unreachable, resulting in a 404 error on GitHub. Can you fix them?

Dscano commented:
@AdvaySingh1,, To pass the DCO check, you must provide a "signature for your code."  A guide is available here: https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md

AdvaySingh1 commented:
> The Gigaflow Project and NextGarch Lab links are working, but all the other links listed in the report are unreachable, resulting in a 404 error on GitHub. Can you fix them?

Some of these repositories will be going public in some more time. Currently, we I'm debugging one of the final steps. Once that's done, the repositories will be make public and the links will work. Is that ok?

Dscano commented:
@annuszulfiqar2021 it is ok for you?

AdvaySingh1 commented:
> @AdvaySingh1, imho you’ve addressed the comments made by Fabian. The link to the GitHub repo for the GSoC project is still missing.

Thanks! Yes I'm still working on the final debugging for that. The repository will be made public soon.

Dscano commented:
> > @AdvaySingh1, imho you’ve addressed the comments made by Fabian. The link to the GitHub repo for the GSoC project is still missing.
> 
> Thanks! Yes I'm still working on the final debugging for that. The repository will be made public soon.

Perfect, thank you for the update.
